recipes-extended/xen/xen-tools_git.bbappend

```diff
@@ -0,0 +1,31 @@
+# Avoid redundant runtime dependency on python3-core
+RDEPENDS:${PN}:remove:class-target = " ${PYTHON_PN}-core" 
+
+require xen-source.inc
+
+FILES:${PN}-test = "\
+    ${libdir}/xen/bin/test-xenstore \
+    ${libdir}/xen/bin/test-resource \
+    ${libdir}/xen/bin/test-paging-mempool\
+"
+
+do_install:append() {
+    rm -f ${D}/${libdir}/xen/bin/init-dom0less
+    rm -f ${D}/${systemd_unitdir}/system/var-lib-xenstored.mount
+    rm -rf ${D}/var
+}
+
+FILES:${PN}-xencommons:remove = "\
+    "${systemd_unitdir}/system/var-lib-xenstored.mount" \
+"
+
+PACKAGECONFIG:append = " xsm"
+
+FILES:${PN}-flask = " /boot/xenpolicy-${XEN_REL}*"
+
+SYSTEMD_SERVICE:${PN}-xencommons:remove = " \
+    var-lib-xenstored.mount \
+"
```

Comment:
What is this? why?

Reply:
https://github.com/xen-troops/meta-xt-rpi5/pull/5#discussion_r1581143151

Reply:
Removed